### PR TITLE
MULE-9498: Timezone attribute in Quartz connector and scheduler module

### PR DESCRIPTION
**Pull request by elrodro83**

### CODE REVIEW

File: modules/schedulers/src/main/resources/META-INF/mule-schedulers.xsd [hunk 1]
```diff
@@ -26,6 +26,13 @@
                         </xsd:documentation>
                     </xsd:annotation>
                 </xsd:attribute>
+                <xsd:attribute name="timeZone" type="xsd:string">
```
Comment (afelisatti) [inline modules/schedulers/src/main/resources/META-INF/mule-schedulers.xsd]: Should this support properties?

Comment (elrodro83) [inline modules/schedulers/src/main/resources/META-INF/mule-schedulers.xsd]: It already does defined as it is. The cron expression is declared in the same way and it is tested to support expressions.

File: modules/schedulers/src/test/java/org/mule/modules/schedulers/cron/CronsTimeZoneSchedulerTest.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        Collection<Scheduler> schedulers = muleContext.getRegistry().lookupScheduler(
+                Schedulers.flowConstructPollingSchedulers("pollWithTimeZone"));
+        
+        assertThat(schedulers, hasSize(1));
+        for (Scheduler scheduler : schedulers)
+        {
+            assertThat(scheduler, instanceOf(CronScheduler.class));
+            // Just can assert that the tx in the config gets to the mule scheduler. To instrospect the quatz object to
+            // check its there would be very complex.
+            assertThat(((CronScheduler) scheduler).getTimeZone(), is(TimeZone.getTimeZone("America/Argentina/Buenos_Aires")));
```
Comment (afelisatti) [inline modules/schedulers/src/test/java/org/mule/modules/schedulers/cron/CronsTimeZoneSchedulerTest.java]: Static import

File: transports/quartz/src/test/resources/quartz-namespace-config-flow.xml [hunk 3]
```diff
@@ -79,6 +79,25 @@
     </flow>
     <!--// END SNIPPET: example-1 -->
 
+    <flow name="testService1Timezone">
+        <description>
+            This configuration will create an inbound event for testService1 at 12
+            noon GMT every day. The event payload
```
Comment (afelisatti) [inline transports/quartz/src/test/resources/quartz-namespace-config-flow.xml]: Shouldn't this be Buenos Aires time?

File: modules/schedulers/src/main/resources/META-INF/mule-schedulers.xsd (hunk 1 above)
Comment (afelisatti) [inline modules/schedulers/src/main/resources/META-INF/mule-schedulers.xsd]: Is this optional?

Comment (elrodro83) [inline modules/schedulers/src/main/resources/META-INF/mule-schedulers.xsd]: Yes, attributes are optional by default in xsd

**Pull request merged**